presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -40,6 +40,18 @@ The following file types are supported for the Hive connector:
 * CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
 * TextFile
 
+In order to enable first-class support for CSV files when using Hive 3.x, you
```

Comment:
I missed that earlier; same applies to Avro tables with `avro_schema_url`.

For ref, we're talking about:
https://github.com/prestosql/presto/blob/ebe16716fb76e4c4883cf7196d9b1087edb51582/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java#L88-L89

Reply:
Updated and ready to merge again.